portfolio/src/main/webapp/style.css

```diff
@@ -1,11 +1,43 @@
+#hufflepuff_stripes {
+    background: repeating-linear-gradient(
+        45deg,
+        gold,
+        gold 100px,
+        black 100px,
+        black 200px
+    );
+}
+
+#ravenclaw_stripes {
+    background: repeating-linear-gradient(
+        45deg,
+        rgb(52, 190, 236),
+        rgb(52, 190, 236) 100px,
+        rgba(143, 66, 22, 0.795) 100px,
+        rgba(143, 66, 22, 0.795) 200px
+    );
+}
+
+#title {
+    font-size: 60px; 
+    font-family: cursive;
+    color: white;
+    text-shadow: -2px 0 orange, 0 2px orange, 2px 0 orange, 0 -2px orange;
```

Comment:
put declarations in alphabetical order: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Added a commit which places the css declarations in alphabetical order.